definitions/assessor_workload_per_day.sqlx

```diff
@@ -59,26 +60,19 @@ GROUP BY
   -- identifies the applications that have a status of assessment in progress
   assessor_workload AS (
 SELECT
-  reference,
-  state,
-  stage,
-  statuses,
-  action_required_by,
+  DISTINCT reference,
   assessor_id,
   DATE(MIN(valid_from)) AS date_actioned_at
 FROM
-  `apply-for-qts-in-england.dataform_production.application_forms_version_afqts` AS application_forms
+   ${ref("application_forms_version_afqts")} AS application_forms
 WHERE
-  stage = 'assessment'
-  AND statuses IN ('assessment_in_progress')
-  AND action_required_by = 'assessor'
+  ((state = 'assessment_in_progress' )

```

Comment:
Do we definitely need this condition on `state`? The new fields should be working fully, so by removing this condition we reduce how much needs to be updated when we remove the old fields.

Reply:
In the last version of this code i omitted the state and then i realised that i could only go back to 29 September when the field changes were made, so to include data from earlier months i included the state. The table I am using is the version table so i am looking at previous states of the data, rather than current state. Once there is no longer interest in those earlier months then the data can be ignored, or i'll exclude in the dashboards or like you have mentioned remove the conditions.

Reply:
Ah yes, that makes sense 👍🏻